spec/features/deploy_spec.rb

```diff
@@ -315,4 +316,14 @@
       expect(webhook).not_to have_been_made
     end
   end
+
+  it 'failed to release due blocked time' do
+    strategy.update!(arguments: strategy.arguments.merge(
+      blocked_time_buckets: ['* 1-23 * * *'],
```

Comment:
Should this be `0-23`? Otherwise it will "fail" (allow a deploy) when tests are run at certain hours of the night.

Reply:
yup, just changed.